pt-BR/active_support_core_extensions.md

```diff
@@ -266,26 +266,26 @@ class Proc
 end
 ```
 
-NOTE: Defined in `active_support/core_ext/kernel/singleton_class.rb`.
+NOTE: Definido em `active_support/core_ext/kernel/singleton_class.rb`.
 
 ### `acts_like?(duck)`
 
-The method `acts_like?` provides a way to check whether some class acts like some other class based on a simple convention: a class that provides the same interface as `String` defines
+O método `acts_like?` fornece um meio para conferir se alguma classe age como alguma outra classe basaeda em uma simples convenção: a classe que fornece a mesma _interface_ é definida como `String`
 
 ```ruby
 def acts_like_string?
 end
 ```
 
-which is only a marker, its body or return value are irrelevant. Then, client code can query for duck-type-safeness this way:
+que é apenas um marcador, seu corpo ou valor de retorno são irrelevantes. Então, o código do cliente pode consultar a tipagem desta forma:
 
 ```ruby
 some_klass.acts_like?(:string)
 ```
 
-Rails has classes that act like `Date` or `Time` and follow this contract.
+Rails possui classes atuantes como `Date` ou `Time` e seguem essa linha.
 
-NOTE: Defined in `active_support/core_ext/object/acts_like.rb`.
+NOTE: Definido em `active_support/core_ext/object/acts_like.rb`.
 
 ### `to_param`
```

Comment:
Aqui que daqui até a linha 499 ainda tá em execução certo? 

Esse tópico é um pouco grande, quer dividir em dois pull requests? 😄 

Pra gente não faz tanta diferença mas aí poderíamos integrar esse e depois o outro. Só lembra de se for fazer isso criar um pull request a partir do branch principal para evitar conflitos

Reply:
Ok, farei as correções e submeterei o restante em outro pull request, até pra facilitar a revisão.

Reply:
Podemos aplicar a sugestão então por enquanto e depois a gente pode encontrar um termo melhor :)

Reply:
Combinado! :]